ui/src/data_explorer/containers/Header.js

```diff
@@ -2,23 +2,24 @@ import React, {PropTypes} from 'react'
 import {withRouter} from 'react-router'
 
 import AutoRefreshDropdown from 'shared/components/AutoRefreshDropdown'
-import TimeRangeDropdown from '../../shared/components/TimeRangeDropdown'
-import SourceIndicator from '../../shared/components/SourceIndicator'
-import GraphTips from '../../shared/components/GraphTips'
+import TimeRangeDropdown from 'shared/components/TimeRangeDropdown'
```

Comment:
these relative --> absolute changes indicate that i'd expect those changes in other files you're modifying in this PR as well